src/components/Amalgamation/ResultingBusinessName.vue

```diff
@@ -152,10 +163,11 @@ export default class ResultingBusinessName extends Mixins(AmalgamationMixin, Nam
     nrNum: string, businessId: string, phone: string, email: string
   ): Promise<NameRequestIF> {
     const nameRequest = await LegalServices.fetchValidContactNr(nrNum, phone, email)
-    if (!nameRequest) throw new Error('Error fetching Name Request')
+    if (!nameRequest) throw new Error('Unable to fetch Name Request.')
```

Comment:
This message will get displayed if the NR can't be found (404), for example due to an invalid NR number.